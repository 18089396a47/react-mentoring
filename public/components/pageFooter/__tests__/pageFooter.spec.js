import React from 'react';
import renderer from 'react-test-renderer';
import PageFooter from '../pageFooter';

jest.mock('react-router-dom', () => ({ Link: 'Link' }));

describe('PageFooter component', () => {
    it('PageFooter should be shown correctly', () => {
        const tree = renderer.create(
            <PageFooter />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
