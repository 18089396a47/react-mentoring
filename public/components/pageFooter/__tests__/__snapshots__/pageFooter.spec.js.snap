// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter component PageFooter should be shown correctly 1`] = `
<footer
  className="page-footer"
>
  <Link
    className="page-footer__link"
    to="/"
  >
    netflixroulette
  </Link>
</footer>
`;
